index.html

```diff
@@ -852,11 +852,21 @@ <h2><dfn>CaptureController</dfn></h2>
           </p>
           <pre class="idl">
             [Exposed=Window, SecureContext]
-            interface CaptureController {
+            interface CaptureController : EventTarget {
               constructor();
               undefined setFocusBehavior(CaptureStartFocusBehavior focusBehavior);
             };
           </pre>
+          <div class="note">
+            <p>
+              {{CaptureController}} does not yet not define any event handlers.
+              As such, it is not required to inherit from {{EventTarget}}.
+              This inheritance was added for the benefit of other specifications
+              that extend {{CaptureController}} with event handler attributes.
+              This inheritance may be removed if these other specifications do
+              not mature successfully.
```

Comment:
Could we use a "Feature at risk:" template/wording for this?

Reply:
Done.